include/zephyr/bluetooth/conn.h

```diff
@@ -1877,6 +1914,21 @@ struct bt_conn_cb {
 				const struct bt_conn_le_subrate_changed *params);
 #endif /* CONFIG_BT_SUBRATING */
 
+#if defined(CONFIG_BT_LE_EXTENDED_FEAT_SET)
+	/** @brief Read all remote features complete event.
+	 *
+	 *  This callback notifies the application that read all remote features
+	 *  procedure of the connection is complete.
```

Comment:
```suggestion
	 *  This callback notifies the application that a 'read all remote features'
	 *  procedure of the connection is completed.
```

Just a suggestion. I want to make it clear that this event fires for each completion of the procedure. I would also like this to refer to `bt_conn_le_read_all_remote_features` and vice versa.